src/lib/components/index/hero.svelte

```diff
@@ -6,15 +6,15 @@
   <div class="container">
     <div class="left">
       <h1 class="brand-header size-xl">
-        We are the largest tech community at
+        We are the largest computer science community at
         <b class="acm-blue">CSUF</b>
       </h1>
 
       <AcmButton text="Join us today!" link="/discord" />
     </div>
 
     <img
-      src="assets/png/hero-illustration.png"
+      src="assets/png/hero-photo_1.png"
```

Comment:
rename this to `hero-photo.png` and remove the old photo

Reply:
u stink